### PR TITLE
deps: bump nibabel, add EDF, remove openneuro-py

### PR DESCRIPTION
**Pull request by sappelhoff**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by sappelhoff** [comment]
@conda-forge-admin, please rerender

**Comment by sappelhoff** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,11 +26,11 @@ requirements:
     - mne-base >=0.24
     - numpy >=1.16.0
     - scipy >=1.5.0
-    - nibabel >=2.2
+    - nibabel >=2.5
     - pybv >=0.6
+    - EDFlib-Python >=1.0.2
```
Comment (sappelhoff) [inline recipe/meta.yaml]: Ah, seems like we need to pull `EDFlib-Python` up to conda-forge first.

Comment (sappelhoff) [reply recipe/meta.yaml]: see https://github.com/conda-forge/staged-recipes/pull/17064

**Review by hoechenberger — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (sappelhoff) [reply recipe/meta.yaml]:
```suggestion
    - edflib-python >=1.0.2
```

**Comment by sappelhoff** [comment]
@conda-forge-admin, please rerender

**Pull request merged**